src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Jaeger\Thrift\Span as JTSpan;
+use Jaeger\Thrift\Tag;
+use Jaeger\Thrift\TagType;
+use OpenTelemetry\SDK\Trace\SpanDataInterface;
+
+class SpanConverter
+{
+    const STATUS_CODE_TAG_KEY = 'op.status_code';
+    const STATUS_DESCRIPTION_TAG_KEY = 'op.status_description';
+
+    /**
+    * Convert span to Jaeger Thrift Span format
+    */
+    public function convert(SpanDataInterface $span): JTSpan
+    {
+        $references = $tags = $logs = [];
+        $startTime = (int) ($span->getStartEpochNanos() / 1e3); // microseconds
+        $duration = (int) (($span->getEndEpochNanos() - $span->getStartEpochNanos()) / 1e3); // microseconds
+        $tags = [
+            self::STATUS_CODE_TAG_KEY => $span->getStatus()->getCode(),
+            self::STATUS_DESCRIPTION_TAG_KEY => $span->getStatus()->getDescription(),
+        ];
+
+        foreach ($span->getAttributes() as $k => $v) {
+            $tags[$k] = $this->sanitiseTagValue($v->getValue());
+        }
+        $tags = $this->buildTags($tags);
+
+        [
+            'traceIdLow' => $traceIdLow,
+            'traceIdHigh' => $traceIdHigh
+        ] = SpanConverter::convertOtlpToJaegerTraceIds($span->getContext()->getTraceID());
+        $spanId = intval($span->getContext()->getSpanID(), 16);
+        $parentSpanId = intval($span->getParentSpanId(), 16);
+
+        //TODO - determine if any of the below commented out code is still needed
```

Comment:
I would think at least events should be, but some real-world testing and comparing with jaeger+zipkin might be a good place to start

Reply:
Yeah this PR isn't meant to acheive spec compliance for the conversion, so maybe I should just take it out for now? It'll be available to reference in other spots as needed later anyways

Reply:
scanning over the [jaeger thrift IDL](https://github.com/jaegertracing/jaeger-idl/blob/main/thrift/jaeger.thrift), the commented code looks like it would work, so I think either uncomment or leave it there and we can get it working in a later PR

Reply:
Makes sense to me, will leave it as is for now